### DISCUSSION TITLE
Changed LAN_SDS_MODE, can't access WebGUI

### DISCUSSION THREAD

**Discussion by Jureek** [Anime4000/RTL960x#39] [Q&A]
Hi all,

Recently I have bought an Opton GP801R and upgraded the FW to latest V2801F_V1.9.0-220425 (thank you for the awesome work Anime4000!). Even after the upgrade I couldn't get my setup to work with my ISP (I am also using Ubiquiti EdgeRouter X-SFP on latest FW). Long story short, I have tried to change LAN_SDS_MODE from 1 to 2 in an attempt to make PPPoE work (the only safety mechanism my ISP uses, they do not check the ONT data). Now I can't access the WebGUI or telnet of the stick. Is connecting by UART the only way to recover web/telnet access to the stick?

**Comment by Anime4000** [comment]
Hi @Jureek 
I sorry to hear that you are lost connection from the stick.

Yes, only way to recover settings is via UART

solder UART pin and plug Stick PCB into SFP slot, make sure SoC CPU are facing up

**Reply by dominiaz** [reply]
Can you modify firmware with script something like "if LAN_SDS_MODE xxx fails after 15mins then it automatically switch to LAN_SDS_MODE 1?"

**Reply by Anime4000** [reply]
it's possible,

you need set SFP+ IP address to 192.168.1.2

then SFP Stick trying to ping SFP+
then failed, revert back to 0 and stick reboot

**Reply by Sergey-SRG** [reply]
> it's possible,
> 
> you need set SFP+ IP address to 192.168.1.2
> 
> then SFP Stick trying to ping SFP+ then failed, revert back to 0 and stick reboot

In what firmware is this implemented?

**Comment by Sergey-SRG** [comment]
Judging by the sniffer data, this is implemented in version 220923!?!?!
![IMG_20230214_104811_215](https://user-images.githubusercontent.com/3891900/218672724-354eff35-1845-4bc4-b3a9-053e83ddbf40.jpg)

**Comment by Sergey-SRG** [comment]
@Anime4000 your answer can be heard, otherwise you don’t want to disassemble and flash again!?!?

**Reply by Anime4000** [reply]
only way to access is by using UART on RTL9601CI
if you using ODI Stick RTL9601D, it has auto nego, no need to worry

**Reply by Sergey-SRG** [reply]
> only way to access is by using UART on RTL9601CI if you using ODI Stick RTL9601D, it has auto nego, no need to worry

I use ODI on SoC RTL9601CI together with CRS326-24G-2S , tell me which mode to set 2.5G for LAN_SDS_MODE?

**Reply by Anime4000** [reply]
I not sure which 2.5G `LAN_SDS_MODE` suitable for `CRS326-24G-2S`, you need test each one and UART for next settings.

RTL9601CI is firmware is buggy, unlike RTL9601D ODI has auto `LAN_SDS_MODE`, no need to worry about bricked
